src/holder.ts

```diff
@@ -60,6 +73,8 @@ export class Holder {
     aud: string,
```

Comment:
I would think more about API for this function. Now 5 inline params feel very confusing.

Reply:
true, nonce, and audience should be an optional input. 
`  async presentVerifiableCredentialSDJWT(
    sdJWT: JWT,
    keyBindingJWTVerifier: KeyBindingVerifier,
    disclosedList: DisclosedList[],
    options?: { nonce?: string; audience?: string },
  ):`

Reply:
I concluded the following would be a better signature, 
KeyBindingVerification can be optional. 

`/**
   * Presents a VC SD-JWT with a key binding JWT.
   * @param sdJWT The SD-JWT to present.
   * @param disclosedList The list of disclosed claims.
   * @param options The options to use.
   * @param options.nonce The nonce to use.
   * @param options.audience The verifier to present the VC SD-JWT to. e.g. https://example.com/verifier
   * @param options.keyBindingVerifierCallbackFn The callback function to verify the key binding JWT with the holder public key.
   * @throws An error if the VC SD-JWT cannot be presented.
   * @returns The VC SD-JWT with the key binding JWT.
   */
  async presentVerifiableCredentialSDJWT(
    sdJWT: JWT,
    disclosedList: DisclosedList[],
    options?: { nonce?: string; audience?: string; keyBindingVerifierCallbackFn?: KeyBindingVerifier },
  ): Promise<{ vcSDJWTWithkeyBindingJWT: JWT; nonce?: string }> {`